docs/conf/portland/2023/writing-day.rst

```diff
@@ -201,3 +201,23 @@ This sprint will give you a few options:
 
 The documentation is written in Sphinx & reStructuredText, but you can try out 
 your own project using any framework, as long as it's open source.
+
+Hank
+~~~~
+
```

Comment:
@kaycebasques simplified the title

Reply:
@kaycebasques, I also reformatted the information below for easier reading/skimming. If you're good with it, I'll add it to the site.

I also may move it to the top of the project list, we're adding new projects at the top. I have two other PRs in so I'll make the change after those are approve.

Thanks for your submission!

Reply:
Thanks for the review. You can edit everything however you see fit.